src/compat_tests.rs

```diff
@@ -18,14 +18,12 @@ fn cargo_out_dir() -> std::path::PathBuf {
     cargo_manifest_dir().join("target")
 }
 
-#[cfg(feature = "signature-meta")]
-use signature::{self, Verifying};
-
 #[cfg(feature = "signature-pgp")]
 mod pgp {
     use super::*;
     use futures::io::AsyncWriteExt;
     use signature::pgp::{Signer, Verifier};
+    use signature::{self, Verifying};
```

Comment:
Should always be exported if we remove `"signature-meta"` - after all it was meant for the case where signature creation/verification is desired, but not the internal implementations.

Reply:
This is in a tests file, it'll just be an unused import.

Reply:
If you're talking about the other module in general, I removed the feature flag and made it public.  The `pgp` submodule is still behind the feature flag.

Reply:
Never mind, I think I see what you mean.